app/__init__.py

```diff
@@ -1,19 +1,10 @@
-from sys import version_info
-
 from configs import Config
 from flask import Flask
 from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
 from flask_limiter import Limiter
 from flask_limiter.util import get_remote_address
 
-try:
-    assert version_info >= (3, 7, 0)
```

Comment:
We use docker now, which should specify the version of Python that we're using. This is now unnecessary

Reply:
Make sense.